src/coreclr/src/tools/Common/Compiler/HardwareIntrinsicHelpers.cs

```diff
@@ -40,6 +40,7 @@ public static bool IsHardwareIntrinsic(MethodDesc method)
             return false;
         }
 
+#if !READYTORUN
         public static bool IsIsSupportedMethod(MethodDesc method)
```

Comment:
Also a bit below at https://github.com/dotnet/runtime/pull/33090/files#diff-2ff7a6ceb92854abf02ee2e4eb4b657eR170, how are SSE4.1, SSE4.2, and POPCNT known at compile time?

Reply:
This is for the non R2R case, where the exact behavior is known (disabled).